conference.js

```diff
@@ -337,6 +338,11 @@ class ConferenceConnector {
             break;
         }
 
+        case JitsiConferenceErrors.CONFERENCE_RESTARTED:
+            APP.store.dispatch(conferenceRestarted(room));
+            APP.store.dispatch(conferenceWillLeave(room));
```

Comment:
The idea was to make sure all the related analytics events for track duration, etc, were being sent before the reload.

Reply:
@jallamsetty1 In the new version of the PR I don't see this part. What changed so that now we  don't need it? Or you figured out that  it was not needed even in the initial version?

Reply:
I missed it, added it back.